src/qt/bitcoingui.cpp

```diff
@@ -567,7 +567,7 @@ void BitcoinGUI::setClientModel(ClientModel *_clientModel)
         connect(_clientModel, &ClientModel::networkActiveChanged, this, &BitcoinGUI::setNetworkActive);
 
         modalOverlay->setKnownBestHeight(_clientModel->getHeaderTipHeight(), QDateTime::fromTime_t(_clientModel->getHeaderTipTime()));
-        setNumBlocks(m_node.getNumBlocks(), QDateTime::fromTime_t(m_node.getLastBlockTime()), m_node.getVerificationProgress(), false);
+        setNumBlocks(m_node.getNumBlocks(), QDateTime::fromTime_t(m_node.getLastBlockTime()), m_node.getVerificationProgress(), false, false);
```

Comment:
```suggestion
        setNumBlocks(m_node.getNumBlocks(), QDateTime::fromTime_t(m_node.getLastBlockTime()), m_node.getVerificationProgress(), /* header */ false, m_node.isInitialBlockDownload());
```

Any reason to not call `m_node.isInitialBlockDownload()` here? This should only be called during the init/de-init sequence, so the overhead should be minimal.

Reply:
Your right. I'll add it there.
I'm currently also trying to get rid of this initial sets in the main GUI thread.